WORKSPACE

```diff
@@ -390,9 +357,10 @@ container_pull(
 # TODO build fedora_with_test_tooling for multi-arch
 container_pull(
     name = "fedora_with_test_tooling",
-    digest = "sha256:ce36d2b4f81b038fba0b61b1bb1ac7f671d47687fb1f9d7ddedd22742cc79dd9",
+    digest = "sha256:14193941e1fe74f2189536263c71479abbd296dc93b75e8a7f97f0b31e78b71e",
     registry = "quay.io",
     repository = "kubevirtci/fedora-with-test-tooling",
+    # tag = "fc32-2108061440",
```

Comment:
line left intentional to present the tag ?
(better to remove total i mean, or at least put in the digest line as a comment, else we might not update it next time ?)